tests/PHPStan/Rules/Generics/data/method-signature-variance.php

```diff
@@ -11,28 +15,175 @@ interface Invariant {
 }
 
 /**
- * @template-covariant T
- * @template-covariant W of \DateTimeInterface
+ * @template K
  */
-class C {
+class A {
```

Comment:
Please do not change existing tests like that, the diff is unreadable. Test the new behaviour in a new file/test case method. Thanks.

Reply:
I'll revert the change and add three more test cases for each variance type.